modules/servodyn/src/ServoDyn.f90

```diff
@@ -2954,7 +2973,7 @@ SUBROUTINE Yaw_UpdateStates( t, u, p, x, xd, z, OtherState, m, ErrStat, ErrMsg )
 
       IF ( .not. OtherState%BegYawMan )  THEN  ! Override yaw maneuver is just beginning (possibly again).
 
-         CALL CalculateStandardYaw(t, u, p, m, YawPosCom, YawRateCom, ErrStat, ErrMsg)
+         CALL CalculateStandardYaw(t, u, p, m, YawPosCom, YawRateCom, OtherState%YawPosComInt, ErrStat, ErrMsg)
```

Comment:
It looks like you could potentially be updating the OtherState%YawPosComInt twice in this call (i.e., taking two steps instead of just one).

Reply:
I missed this bit of logic.  The second call to `CalculateStandadYaw` is during an override yaw manuever and should only be called once while `OtherState%BegYawMan` is false.  If we are doing an over-ride of the yaw from the DLL, I would expect the DLL commanded yaw should be completely out of the loop then, so maybe the first first `CalculateStandardYaw should be in an `else` condition on `t >= p%TYawManS`.